### PR TITLE
Add current limit

### PR DESCRIPTION
**Pull request by get-itips**
closes https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/10256

Cross-checked with Lukas Barton

### CODE REVIEW

File: Teams/trusted-organizations-external-meetings-chat.md [hunk 1]
```diff
@@ -86,6 +86,8 @@ You can specify which domains are allowed or which domains are blocked. If you s
 
 > [!NOTE]
 > People from blocked domains can still join meetings anonymously if anonymous access is allowed. To learn more, see [Manage anonymous participant access to Teams meetings](anonymous-users-in-meetings.md).
+> 
+> Current limit for allow or block specific domains list is 4000 domains.
```
Comment (JohanFreelancer9) [inline Teams/trusted-organizations-external-meetings-chat.md:90]:
```suggestion
> The current limit for an allow or block specific domains list is 4,000 domains.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@MikePlumleyMSFT 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by MikePlumleyMSFT** [comment]
There's an internal discussion on what limits to document for this feature. Closing this for now.

**Pull request closed without merging**